src/Module.Server/Rewards/CrpgRewardServer.cs

```diff
@@ -410,7 +584,7 @@ private IList<CrpgUserDamagedItem> BreakItems(CrpgPeer crpgPeer, float roundDura
     }
 
     private void SendRewardToPeers(IList<UpdateCrpgUserResult> updateResults,
-        Dictionary<int, CrpgPeer> crpgPeerByUserId, HashSet<PlayerId> valorousPlayerIds)
+        Dictionary<int, CrpgPeer> crpgPeerByUserId, HashSet<PlayerId> valorousPlayerIds, Dictionary<int, int> netCompensationByCrpgUserId)

```

Comment:
NIT: Small inconsistency here, one dictionary is named `ByUserId` and the other `ByCrpgUserId` I would go with the former everywhere in the class.